tasks/setup-docker-group.yml

```diff
@@ -22,8 +24,8 @@
 
     # See https://stackoverflow.com/a/44753457
     - name: Reset ansible connection after group changes (Ansible >= 2.3.0).
-      include: reset-connection.yml
-      static: false  # we need to force dynamic inclusion here to avoid 'reset_connection' to be
+      include_tasks: reset-connection.yml
+      # we need to force dynamic inclusion here to avoid 'reset_connection' to be
```

Comment:
Looks like this comment needs to go too, then? And does this still work?

Reply:
@rwos So I left the comment because we use `include_tasks` which is dynamic, vs. `import_tasks` (static).

As for if this still works, I'll add a test suite in another PR before I want to merge this.

Reply:
aaaaah, missed that 👍